### ISSUE TITLE
Cannot publish using `ray publish`

### ISSUE THREAD

**Issue by Sheraff** [raycast/extensions#9348]
### Description

# First issue

Trying to contribute to the [Port Manager](https://www.raycast.com/lucaschultz/port-manager) extension, the `ray publish` command is still looking at an old extension of mine I haven't touched in over a year.

```shell
fork extensions repo: could not get the latest changes. Head to https://github.com/Sheraff/extensions-urban-dictionary, select the Sync fork dropdown menu above the list of files, and then click Update branch. Once you've done that, try running this command again

Error: Error Body: {"message":"There are merge conflicts","documentation_url":"https://docs.github.com/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository"}
```

Thankfully this was fixable by going syncing my repo (thank you for a very explicit error message that includes a solution!). However it would be nice if it only looked at the extension I'm actually trying to publish when running `ray publish` from a specific extension folder.

# Second issue

Since my first commit was made through the VSCode UI, i was able to add `"` characters without worrying about escaping them, though it looks like whatever the `ray publish` command is doing, it's not handling them well. I tried adding a second commit without `"` but it's still reading both, and so the command fails.

```shell
failed running git command 'git add "/Users/pellet/.config/raycast/public-extensions-fork/extensions/port-manager" && git commit -m "Update port-manager extension

- Add known-ports, making it easier to find commonly used processes
- Add "known ports", making it easier to find commonly used processes"', exit status 1

error: pathspec 'ports, making it easier to find commonly used processes' did not match any file(s) known to git
```

Thankfully this too is fixable, and there is documentation on how to rename a commit here https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message#amending-older-or-multiple-commit-messages. However it would be nice if the `ray publish` command did not fail when there is a `"` character in a commit message.

### Steps To Reproduce

Not sure how to reproduce Issue 1. Probably 
- create extension using a repo outside of `github.com/raycast`
- have other people contribute to the extension on `github.com/raycast`
- create a 2nd extension repo (using the "Fork Extension" raycast command)
- run `ray publish` from the 2nd extension folder

For issue 2
- create an extension repo (using the "Fork Extension" raycast command)
- commit to it with a message that includes at least 1 `"` character
- run `ray publish` from that extension's folder

**Comment by mathieudutour** [comment]
> Trying to contribute to the [Port Manager](https://www.raycast.com/lucaschultz/port-manager) extension, the ray publish command is still looking at an old extension of mine I haven't touched in over a year.

It isn't, it's looking at the fork that you have on your account (which has to be synced with this one).

> Since my first commit was made through the VSCode UI, i was able to add " characters without worrying about escaping them, though it looks like whatever the ray publish command is doing, it's not handling them well

Ah yep, I thought I fixed it already but I'll have a second look

**Comment by Sheraff** [comment]
> It isn't, it's looking at the fork that you have on your account (which has to be synced with this one).

Right! Thanks. It wasn't very clear to me because the old way used to be that you have to clone the entire repo, and not it's creating a small contained one for you, and on publish will look for the complete one.
